assets/react/v3/entries/course-builder/components/additional/meeting/GoogleMeetCard.tsx

```diff
@@ -141,7 +141,7 @@ const GoogleMeetMeetingCard = ({ data, topicId }: GoogleMeetMeetingCardProps) =>
         triggerRef={deleteRef}
         closePopover={noop}
         maxWidth="258px"
-        title={`Delete meeting "${post_title}"`}
+        title={`${__('Delete meeting')} "${post_title}"`}
```

Comment:
Use sprintf helper for dynamic translation
```
sprintf( __( 'Delete metting %s', 'tutor' ), title )
```

Reply:
Updated